2017-11-09 11:01:00.730 [info] <0.560.0> Application lager started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.730 [info] <0.560.0> Application ranch started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.730 [info] <0.560.0> Application ranch_proxy_protocol started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.730 [info] <0.560.0> Application cowboy started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.730 [info] <0.560.0> Application rabbit_common started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.730 [info] <0.560.0> Application amqp_client started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.730 [info] <0.560.0> Application pampa_pro started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.732 [info] <0.560.0> Application sasl started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.733 [info] <0.560.0> Application runtime_tools started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:03:43.764 [error] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:9: Error: record amqp_params_direct undefined
/home/pampa/pampa_pro/src/pampa_pro_app.erl:9: Warning: variable 'Connection' is unused
2017-11-09 11:04:14.138 [warning] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:10: Warning: variable 'Connection' is unused
/home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled with 1 warnings
2017-11-09 11:04:14.154 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 11:04:53.624 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 11:04:53.642 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 11:05:21.950 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 11:05:55.313 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 11:05:55.326 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 11:06:15.561 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 11:06:28.792 [info] <0.560.0> Application lager started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:06:28.793 [info] <0.560.0> Application ranch started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:06:28.793 [info] <0.560.0> Application ranch_proxy_protocol started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:06:28.793 [info] <0.560.0> Application cowboy started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:06:28.793 [info] <0.560.0> Application rabbit_common started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:06:28.793 [info] <0.560.0> Application amqp_client started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:06:28.794 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}}) line 239
2017-11-09 11:06:28.794 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}}) line 239
2017-11-09 11:06:28.795 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}}) line 239 in context child_terminated
2017-11-09 11:06:28.795 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:06:28.795 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@127.0.0.1.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510196788794},{node,'pampa_pro@127.0.0.1'},{user_who_performed_action,none}]],[]},{...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:06:28.795 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@127.0.0.1.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510196788794},{node,'pampa_pro@127.0.0.1'},{user_who_performed_action,none}]],[]},{...},...]}}}},...},...],...},...]},...}
2017-11-09 11:07:28.418 [info] <0.560.0> Application lager started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.418 [info] <0.560.0> Application ranch started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.419 [info] <0.560.0> Application ranch_proxy_protocol started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.419 [info] <0.560.0> Application cowboy started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.419 [info] <0.560.0> Application rabbit_common started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.419 [info] <0.560.0> Application amqp_client started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.435 [info] <0.560.0> Application pampa_pro started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.437 [info] <0.560.0> Application sasl started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.438 [info] <0.560.0> Application runtime_tools started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:56.481 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 11:07:56.493 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 11:08:09.503 [info] <0.560.0> Application lager started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:08:09.503 [info] <0.560.0> Application ranch started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:08:09.503 [info] <0.560.0> Application ranch_proxy_protocol started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:08:09.504 [info] <0.560.0> Application cowboy started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:08:09.504 [info] <0.560.0> Application rabbit_common started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:08:09.504 [info] <0.560.0> Application amqp_client started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:08:09.505 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}}) line 239
2017-11-09 11:08:09.505 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}}) line 239
2017-11-09 11:08:09.505 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}}) line 239 in context child_terminated
2017-11-09 11:08:09.506 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:08:09.506 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@127.0.0.1.1.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510196889505},{node,'pampa_pro@127.0.0.1'},{user_who_performed_action,none}]],[]},{...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:08:09.506 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@127.0.0.1.1.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510196889505},{node,'pampa_pro@127.0.0.1'},{user_who_performed_action,none}]],[]},{...},...]}}}},...},...],...},...]},...}
2017-11-09 11:08:59.868 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 11:08:59.868 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 11:08:59.868 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 11:08:59.869 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 11:08:59.869 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 11:08:59.869 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 11:08:59.869 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 11:08:59.870 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 11:08:59.870 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239 in context child_terminated
2017-11-09 11:08:59.870 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:08:59.870 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510196939869},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:08:59.870 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510196939869},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...}
2017-11-09 11:11:25.877 [info] <0.560.0> Application lager started on node pampa_pro@VM_0_14_centos
2017-11-09 11:11:25.878 [info] <0.560.0> Application ranch started on node pampa_pro@VM_0_14_centos
2017-11-09 11:11:25.878 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@VM_0_14_centos
2017-11-09 11:11:25.878 [info] <0.560.0> Application cowboy started on node pampa_pro@VM_0_14_centos
2017-11-09 11:11:25.878 [info] <0.560.0> Application rabbit_common started on node pampa_pro@VM_0_14_centos
2017-11-09 11:11:25.878 [info] <0.560.0> Application amqp_client started on node pampa_pro@VM_0_14_centos
2017-11-09 11:11:25.878 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}}) line 239
2017-11-09 11:11:25.879 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}}) line 239
2017-11-09 11:11:25.879 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}}) line 239 in context child_terminated
2017-11-09 11:11:25.879 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:11:25.879 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@VM_0_14_centos.3.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510197085878},{node,pampa_pro@VM_0_14_centos},{user_who_performed_action,none}]],...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:11:25.880 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@VM_0_14_centos.3.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510197085878},{node,pampa_pro@VM_0_14_centos},{user_who_performed_action,none}]],...},...]}}}},...},...],...},...]},...}
2017-11-09 11:13:25.720 [info] <0.560.0> Application lager started on node pampa_pro@VM_0_14_centos
2017-11-09 11:13:25.720 [info] <0.560.0> Application ranch started on node pampa_pro@VM_0_14_centos
2017-11-09 11:13:25.720 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@VM_0_14_centos
2017-11-09 11:13:25.720 [info] <0.560.0> Application cowboy started on node pampa_pro@VM_0_14_centos
2017-11-09 11:13:25.720 [info] <0.560.0> Application rabbit_common started on node pampa_pro@VM_0_14_centos
2017-11-09 11:13:25.721 [info] <0.560.0> Application amqp_client started on node pampa_pro@VM_0_14_centos
2017-11-09 11:13:25.721 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}}) line 239
2017-11-09 11:13:25.721 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}}) line 239
2017-11-09 11:13:25.721 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}}) line 239 in context child_terminated
2017-11-09 11:13:25.721 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:13:25.722 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@VM_0_14_centos.1.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510197205720},{node,pampa_pro@VM_0_14_centos},{user_who_performed_action,none}]],...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:13:25.722 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@VM_0_14_centos.1.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510197205720},{node,pampa_pro@VM_0_14_centos},{user_who_performed_action,none}]],...},...]}}}},...},...],...},...]},...}
2017-11-09 11:16:16.748 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 11:16:16.748 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 11:16:16.748 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 11:16:16.748 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 11:16:16.749 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 11:16:16.749 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 11:16:16.750 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 11:16:16.750 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 11:16:16.751 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239 in context child_terminated
2017-11-09 11:16:16.751 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:16:16.751 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510197376749},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:16:16.751 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510197376749},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...}
